orchestrator-salt/src/main/resources/salt/salt/top.sls

```diff
@@ -56,6 +56,10 @@ base:
     - match: compound
     - gateway.knox
 
+  'G@roles:idbroker':
```

Comment:
We will need the same in the pillar/top.sls file otherwise the idbroker node won't be able to access the pillar data for idbroker.

Reply:
I don't think we have idbroker role today. Example for knox: com.sequenceiq.cloudbreak.core.bootstrap.service.host.GrainPropertiesService#addKnoxRoleForHosts